bsb/morphologies/__init__.py

```diff
@@ -513,8 +513,11 @@ def rotate(self, rot, center=None):
         Point rotation
 
         :param rot: Scipy rotation
-        :type: :class:`scipy.spatial.transform.Rotation`
+        :type rot: :class:`scipy.spatial.transform.Rotation`
```

Comment:
```suggestion
        :type rot: scipy.spatial.transform.Rotation
```

the class role is inferred by sphinx